### DISCUSSION TITLE
Option to turn off security arrests

### DISCUSSION THREAD

**Discussion by BrogueMan** [FakeFishGames/Barotrauma#5739] [Ideas and suggestions]
I think having an option to turn off security arrests would be really useful. It is really annoying to miss a enemy and shoot your crew and then have every single security officer stop what they were doing just to arrest you. It makes sense to have it as an option if you want it. But in single player it can ruin games. I have been arrested while fixing a small respawn shuttle because, there was no way to fix it with out damaging the bots within.

https://user-images.githubusercontent.com/64657392/117700046-25579780-b18b-11eb-9f86-b1d868fb889e.mp4

**Comment by PrincessMuu-Muu** [comment]
This mechanic does annoy me, but there are ways around it. In the clip for example, the player empties half a mag into the back of an NPC, that was no one's fault but the player. 

Keep in mind, if this didn't happen many NPCs would be senselessly murdered on multiplayer by other players. (I feel as though this was a countermeasure by the devs to try and stop people from attacking NPCs.) 

As for loopholes... If you don't mind enabling cheats, you can simply run the kill command on the security NPCs and then "spawn human cursor" to replace the NPC.

I personally want them to add an option that lets the host hand pick the professions of the NPC crew.
